packages/fscomponents/src/components/MultiCarousel/MultiCarouselProps.ts

```diff
@@ -18,8 +18,12 @@ export interface MultiCarouselProps<ItemT> {
   itemStyle?: any;
   itemUpdated?: (oldItem: ItemT, newItem: ItemT, index: number, changed: () => void) => void;
   onSlideChange?: (data: SlideChangeEvent) => void;
+  nextArrowContainerStyle?: any;
```

Comment:
You can make these four types `StyleProp<ViewStyle>` so they're typed correctly coming in. I know we didn't do that with the other style props in this component but might as well start doing it right. :)